### PR TITLE
like inbox and display like

### PR DESCRIPTION
**Pull request by Weida-W**

### CODE REVIEW

File: frontend/src/components/PostDisplay/index.js [hunk 1]
```diff
@@ -82,59 +97,61 @@ export default class PostDisplay extends React.Component {
...
+      //           (item) => item.value === this.props.authorID
+      //         ),
+      //         1
+      //       ),
+      //     };
+      //   },
+      //   () => {
+      //     console.log(this.state.likeslist);
+      //   }
+      // );
```
Comment (xqiu1) [inline frontend/src/components/PostDisplay/index.js]: You can delete these if no use.

File: backend/presentation/views.py [hunk 2]
```diff
@@ -146,7 +146,10 @@ def getInboxLike(request, author_id):
     author = Author.objects.get(id=au_id)
     inbox = Inbox.objects.get(author=author)
     like_list = []
+    print(inbox.items)
     for each in inbox.items:
         if each["type"] == "Like":
             like_list.append(each)
+    print("-------")
+    print(like_list)
```
Comment (xqiu1) [inline backend/presentation/views.py]:
```suggestion
    print("-------")
    print(like_list)
```
Delete debugging codes

File: backend/presentation/views.py [hunk 3]
```diff
@@ -146,7 +146,10 @@ def getInboxLike(request, author_id):
     author = Author.objects.get(id=au_id)
     inbox = Inbox.objects.get(author=author)
     like_list = []
+    print(inbox.items)
```
Comment (xqiu1) [inline backend/presentation/views.py:149]:
```suggestion
    print(inbox.items)
```

File: backend/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (xqiu1) [inline backend/yarn.lock]: What's this?

File: backend/.inputrc [hunk 5]
```diff
@@ -0,0 +1 @@
+set completion-ignore-case on set show-all-if-ambiguous on TAB: menu-complete
```
Comment (xqiu1) [inline backend/.inputrc]: What's this?

File: frontend/src/components/PostDisplay/index.js [hunk 6]
```diff
@@ -85,8 +140,9 @@ export default class PostDisplay extends React.Component {
     let params = this.props.rawPost;
     params.authorID = this.state.authorID;
     params.visibility = "FRIENDS";
-    params.title = "Shared " + params.authorName + "\'s  \"" + params.title + "\"";
-    console.log("hhh: " + JSON.stringify(params) );
+    params.title =
+      "Shared " + params.authorName + "'s  \"" + params.title + '"';
+    console.log("hhh: " + JSON.stringify(params));
```
Comment (xqiu1) [inline frontend/src/components/PostDisplay/index.js]:
```suggestion
    console.log("hhh: " + JSON.stringify(params));
```
same here

File: frontend/src/components/PostDisplay/index.js [hunk 7]
```diff
@@ -127,61 +183,61 @@ export default class PostDisplay extends React.Component {
...
+        actor: this.props.authorID,
+        object: this.props.postID,
+        summary: "I like you post!",
+        context: this.props.postID,
+      };
+
+      sendLikes(params).then((response) => {
+        if (response.status === 200) {
+          message.success("Likes sent!");
+          console.log("like");
```
Comment (xqiu1) [inline frontend/src/components/PostDisplay/index.js]:
```suggestion
          console.log("like");
```

File: frontend/src/components/PostDisplay/index.js [hunk 8]
```diff
@@ -127,61 +183,61 @@ export default class PostDisplay extends React.Component {
...
 
-  clickLikeComment = () => {};
+      let params = {
+        postID: this.props.postID,
+        actor: this.props.authorID,
+        object: this.state.comments[0].commentid,
+        summary: "I like you comment!",
+        context: this.props.postID,
+      };
+      console.log("params", params);
```
Comment (xqiu1) [inline frontend/src/components/PostDisplay/index.js]:
```suggestion
      console.log("params", params);
```

**Review by xqiu1 — CHANGES_REQUESTED** [review]

**Review by xqiu1 — APPROVED** [review]
Fix remaining things

**Review by AcidCannon — APPROVED** [review]
Good.

**Comment by AcidCannon** [comment]
Should this merge into master branch?

**Pull request merged**